js/scripts/zone_loaders/village_zone_loaders.js

```diff
@@ -136,45 +139,45 @@ const loadVillageField = () => {
 
                 for (let i = 0; i < 5; i += 0.5) {
                     i += .5;
-                    GAME.addEntity(new Decoration(Decoration.DECORATIONS.flowers.TALL_PURPLE_FLOWER_3, Vector.blockToWorldSpace(new Vector(x + i, ZONE.MAX_BLOCK.y - 1))), 0);
+                    GAME.addEntity(new Decoration(Decoration.DECORATIONS.flowers.TALL_PURPLE_FLOWER_3, Vector.blockToWorldSpace(new Vector(x + i, 20))), 0);
                     i += .4;
-                    GAME.addEntity(new Decoration(Decoration.DECORATIONS.flowers.PRIDE_FLOWER_4, Vector.blockToWorldSpace(new Vector(x + i, ZONE.MAX_BLOCK.y - 1))), 0);
+                    GAME.addEntity(new Decoration(Decoration.DECORATIONS.flowers.PRIDE_FLOWER_4, Vector.blockToWorldSpace(new Vector(x + i, 20))), 0);
                     i += .35;
 
-                    GAME.addEntity(new Decoration(Decoration.DECORATIONS.flowers.MED_RED_FLOWER_3, Vector.blockToWorldSpace(new Vector(x + i, ZONE.MAX_BLOCK.y - 1))), 0);
+                    GAME.addEntity(new Decoration(Decoration.DECORATIONS.flowers.MED_RED_FLOWER_3, Vector.blockToWorldSpace(new Vector(x + i, 20))), 0);
```

Comment:
It may be good to keep the position in terms of some named value, e.g. `ZONE.MAX_BLOCK.y - 5` (or whatever value makes sense) in order to keep this more maintainable.

Reply:
What do you mean exactly? Are you speaking of the order in which entities are added to the game? Can you elaborate?

Reply:
Previously, the y positions of these decorations was set to `ZONE.MAX_BLOCK.y - 1` and you changed it to `20`. I am saying that defining the y position in terms of `ZONE.MAX_BLOCK.y` or something similar, like what was done before, may be better than simply hardcoding it.

Reply:
Gotcha! Good idea. Yes I am generally against hard coding values as I have done and Can adjust that.